templates/taxi/driver_list.html

```diff
@@ -1,30 +1,33 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
-    </h1>
+  <h1>Driver List
+    <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
```

Comment:
Code Style issue: According to the checklist, URL identification parameter `name` should use `-` to split words, not `_`. Therefore, `driver-create` should be `driver-create`.